chain/epoch-manager/src/adapter.rs

```diff
@@ -956,6 +963,31 @@ impl EpochManagerAdapter for EpochManagerHandle {
         }
     }
 
+    fn verify_chunk_endorsement(
```

Comment:
Would it be easy to add a simple test just for this function?

Reply:
Done